tinywallet/tinywallet/app.py

```diff
@@ -390,86 +367,22 @@ def signal_(self, s):
         cb, sigA, a, sigK, k = s
         cb(*sigA, *a, **sigK, **k)
 
-    def setWallet(self, wallet):
+    def setWallet_(self, wallet):
```

Comment:
Why the trailing underscore?

Reply:
It's routed through a Qt signal because the `AssetScreen` must be constructed in the main thread.